Mathlib/RingTheory/Polynomial/Chebyshev.lean

```diff
@@ -801,6 +807,34 @@ theorem add_one_mul_T_eq_poly_in_U (n : ℤ) :
   linear_combination (norm := (push_cast; ring_nf))
     h₁ + (n + 2) * h₂
 
+theorem add_one_mul_self_mul_T_eq_poly_in_T (n : ℤ) :
+    ((n + 1) * n : R[X]) * (T R (n + 1)) =
+    (n : R[X]) * X * derivative (T R (n + 1)) - (n + 1 : R[X]) * derivative (T R n) := by
+  have h := T_eq_X_mul_U_sub_U (R := R) (n - 1)
+  rw [T_derivative_eq_U, T_derivative_eq_U]
+  linear_combination (norm := (push_cast; ring_nf))
+    (n + 1) * n * h
+
+theorem one_sub_X_sq_mul_derivative_derivative_T_eq_poly_in_T (n : ℤ) :
+    (1 - X ^ 2) * derivative^[2] (T R n) = X * derivative (T R n) - (n ^ 2 : R[X]) * T R n := by
+  have h₁ := congr_arg derivative <| one_sub_X_sq_mul_derivative_T_eq_poly_in_T (R := R) (n - 1)
+  simp only [derivative_sub, derivative_mul, derivative_X, derivative_one, derivative_X_pow,
+    C_eq_natCast, sub_add_cancel, Int.cast_sub, Int.cast_one, derivative_intCast] at h₁
+  have h₂ := add_one_mul_self_mul_T_eq_poly_in_T (R := R) (n - 1)
+  rw [Function.iterate_succ, Function.iterate_one, Function.comp_apply]
+  linear_combination (norm := (push_cast; ring_nf))
+    h₁ + h₂
+
+theorem one_sub_X_sq_mul_derivative_derivative_U_eq_poly_in_U (n : ℤ) :
+    (1 - X ^ 2) * derivative^[2] (U R n) =
+      3 * X * derivative (U R n) - ((n + 2) * n : R[X]) * U R n := by
+  have h := congr_arg derivative <| add_one_mul_T_eq_poly_in_U (R := R) n
+  simp only [derivative_add, derivative_sub, derivative_mul, derivative_X, derivative_one,
+    derivative_X_pow, derivative_intCast, C_eq_natCast, T_derivative_eq_U] at h
+  rw [Function.iterate_succ, Function.iterate_one, Function.comp_apply]
+  linear_combination (norm := (push_cast; ring_nf))
+    h
```

Comment:
```suggestion
  linear_combination (norm := (push_cast; ring_nf)) h
```
I personally find this line break a bit weird but if you are attached to this style I don't particular want to block this PR because of it.

Reply:
(and the other ones)